.dockerignore

```diff
@@ -1,2 +1,7 @@
 .git*
 .vagrant
+/build/
+*~
+*.o
+*.a
+*.so
```

Comment:
Are these necessary? Wouldn't all of these files be in /build?

Reply:
For cmake, yes they should all be in the build folder. Autotools creates object files directly into the source folder however. This could eventually be removed along with the autotools build system.

`*~` is for temporary files produced by emacs.